src/current/v22.1/import-performance-best-practices.md

```diff
@@ -5,28 +5,89 @@ toc: true
+
+- [Choose a performant import format](#choose-a-performant-import-format)
+- [Split your data into multiple files](#split-your-data-into-multiple-files)
+- [Sort your data](#sort-your-data)
 
 {{site.data.alerts.callout_info}}
-If the import size is small, then you do not need to do anything to optimize performance. In this case, the import should run quickly, regardless of the settings.
+If the import size is less than 100 GiB, then you do not need to do anything to optimize performance. For such small datasets, the import should run quickly, regardless of the settings.
 {{site.data.alerts.end}}
 
+## Choose a performant import format
+
+Different import file formats do not have the same performance due to the way they are processed by CockroachDB. The fastest import file formats are:
+
+1. [`CSV`](migrate-from-csv.html) or [`DELIMITED DATA`](import-into.html) (both have about the same import performance).
+1. [`AVRO`](migrate-from-avro.html).
+
+We recommend formatting your import files as `CSV` or `AVRO`. These formats can be processed in parallel by all the nodes in the cluster, which increases performance. To import in these formats, use [`IMPORT INTO`](import-into.html).
+
+{% include {{ page.version.version }}/import-table-deprecate.md %}
+
+### Import the schema separately from the data
+
+When importing into a new table split your dump data into two files:
+
+1. A SQL file containing the table schema.
+1. A CSV, delimited, or AVRO file containing the table data.
+
+Convert the schema-only file using the [Schema Conversion Tool](../cockroachcloud/migrations-page.html). The Schema Conversion Tool automatically creates a new {{ site.data.products.serverless }} database with the converted schema. {% include cockroachcloud/migration/sct-self-hosted.md %}
+
+Then use the [`IMPORT INTO`](import-into.html) statement to import the CSV data into the newly created table:
+
+~~~ sql
+> IMPORT INTO customers (id, name)
+CSV DATA (
+    'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/customers.csv'
+);
+~~~
+
+This method has the added benefit of alerting on potential issues with the import sooner; that is, you will not have to wait for the file to load both the schema and data just to find an error in the schema.
+
+### Import into a schema with secondary indexes
+
+When importing data into a table with secondary indexes, the import job will ingest the table data and required secondary index data concurrently. This may result in a longer import time compared to a table without secondary indexes. However, this typically adds less time to the initial import than following it with a separate pass to add the indexes. As a result, importing tables with their secondary indexes is the default workflow, an effective strategy for most migrations.
```

Comment:
```suggestion
When importing data into a table with [secondary indexes](schema-design-indexes.html), the import job will ingest the table data and required secondary index data concurrently. This may result in a longer import time compared to a table without secondary indexes. However, this typically adds less time to the initial import than following it with a separate pass to add the indexes. As a result, importing tables with their secondary indexes is the default workflow, an effective strategy for most migrations.
```